.github/workflows/mvn-verify-check.yml

```
@@ -30,7 +30,45 @@ env:
     -Dmaven.scaladoc.skip
 
 jobs:
+  cache-dependencies:
+    runs-on: ubuntu-latest
+    outputs:
+      dailyCacheKey: ${{ steps.generateCacheKey.outputs.dailyCacheKey }}
+    steps:
+      - uses: actions/checkout@v3 # refs/pull/:prNumber/merge
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - name: populate-daily-cache
+        run: |
```

Once you move the the cache steps above `name: populate-daily-cache`
```suggestion
        if: steps.cache.outputs.cache-hit != 'true'
        run: |
```